spec/components/stylesheet/manager_spec.rb

```diff
@@ -135,6 +135,67 @@ def manager(theme_id = nil)
       )
     end
 
+    context "stylesheet order " do
+      let(:z_child_theme) { Fabricate(:theme, component: true, name: "ze component").tap { |z|
+        z.set_field(target: :desktop, name: "scss", value: ".child_desktop{.scss{color: red;}}")
+        z.save!
+      }}
+
+      let(:remote) { RemoteTheme.create!(remote_url: "https://github.com/org/remote-theme1") }
+
+      let(:child_remote) { Fabricate(:theme, remote_theme: remote, component: true).tap { |t|
+        t.set_field(target: :desktop, name: "scss", value: ".child_desktop{.scss{color: red;}}")
+        t.save!
+      }}
+
+      it 'output remote child, then sort children alphabetically, then local parent' do
+        theme.add_relative_theme!(:child, z_child_theme)
+        theme.add_relative_theme!(:child, child_remote)
+
+        manager = manager(theme.id)
+        hrefs = manager.stylesheet_details(:desktop_theme, 'all')
+
+        parent = hrefs.select { |href| href[:theme_id] == theme.id }.first
+        childA = hrefs.select { |href| href[:theme_id] == child_theme.id }.first
+        childZ = hrefs.select { |href| href[:theme_id] == z_child_theme.id }.first
+        childR = hrefs.select { |href| href[:theme_id] == child_remote.id }.first
```

Comment:
We're missing camelCase and snake_case for the variables here. Usually we default to snake_case for variable names in Ruby.